core/client/src/main/java/alluxio/client/file/policy/LocalFirstPolicy.java

```diff
@@ -60,6 +60,12 @@ public WorkerNetAddress getWorkerForNextBlock(Iterable<BlockWorkerInfo> workerIn
   }
 
   @Override
+  public WorkerNetAddress getWorkerForBlock(Iterable<BlockWorkerInfo> workerInfoList, long blockId,
```

Comment:
To avoid needing the change the API often to include more bits of info, I suggest adding an options parameter.